.golangci.yml

```diff
@@ -103,5 +104,11 @@ issues:
     - path: _test\.go
       linters:
         - golint
+    - path: localconnectionmonitor/local_connection_event\.go
```

Comment:
can we fix duplicaiton?

Reply:
No, because there is a need to cast map from abstract type to package-special and the same for enum values.